addons/point_of_sale/static/src/app/screens/product_screen/product_screen.js

```diff
@@ -356,7 +356,7 @@ export class ProductScreen extends Component {
             list = fuzzyLookup(
                 this.searchWord,
```

Comment:
```suggestion
                unaccent(this.searchWord, false),
```
Or put the unaccent call directly in the `get searchWord()`.